java/ingest/ingest-status-store/src/test/java/sleeper/ingest/status/store/job/QueryIngestJobStatusByPeriodIT.java

```diff
@@ -54,14 +54,14 @@ public void shouldReturnIngestJobsInPeriod() {
     public void shouldExcludeIngestJobOutsidePeriod() {
         // Given
         IngestJob job = jobWithFiles("file");
-        Instant startedTime = Instant.now();
+        Instant startedTime = Instant.parse("2023-01-03T14:50:00.001Z");
 
         // When
         store.jobStarted(DEFAULT_TASK_ID, job, startedTime);
 
         // Then
-        Instant periodStart = Instant.now().plus(Period.ofDays(1));
-        Instant periodEnd = periodStart.plus(Period.ofDays(1));
+        Instant periodStart = Instant.parse("2023-01-01T14:00:00.001Z");
+        Instant periodEnd = Instant.parse("2023-01-02T14:00:00.001Z");
```

Comment:
These can be moved to the Given part next to the startedTime so we can compare them more easily.